textual-practices.csl

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB">
+  <info>
+    <title>Textual Practices</title>
+    <title-short>TextPrac</title-short>
+    <id>http://www.zotero.org/styles/textual-practices</id>
+    <link href="http://www.zotero.org/styles/textual-practices" rel="self"/>
+    <author>
```

Comment:
Could you add here also the style you used as a template.
Could you also add here a link to the style guidelines using rel="documentation"?